youtube_dl/extractor/svt.py

```diff
@@ -360,11 +364,26 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, playlist_id)
 
+        redux_state = self._parse_json(
+            self._search_regex(
+                r'\.\s*reduxState=\s*({.*})',
+                webpage,
+                'reduxState'),
+            None,
+            transform_source=js_to_json)
```

Comment:
Remove superfluous verbosity.

Reply:
I apologize, but your intent eludes me again, for I do not understand what you mean by *superfluous verbosity*.